src/network/hobbits/peer.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @module network/libp2p
+ */
+
+import {
+  Hello, Status, RequestBody, ResponseBody
+} from "../../types";
+import {Method} from "../../constants";
+import {HobbitsConnectHandler} from "./hobbitsConnectHandler";
+import net from "net";
+import {HobbitsUri} from "./hobbitsUri";
+
+
+export class Peer {
+  public hobbitsUri: HobbitsUri;
```

Comment:
@wemeetagain have you suggested me to change `hobbitsUri` to `peerInfo` here? Shouldn't using `hobbitsUri` will be related as `peerInfo` is not related to the `hobbits` protocol?

Reply:
Its related, but we don't want to expose hobbitsUris to the outside. We're using PeerInfos to identify peers.
If we don't store a Peerinfo (or at least a PeerId) here, we will have to be creating new, different PeerIds every time we generate PeerInfos.